go/api/config_test.go

```diff
@@ -0,0 +1,126 @@
+// Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+import (
+	"fmt"
```

Comment:
What it is used for?

Reply:
Its used in TestNodeAddress and TestRedisCredentials. Both of these tests use t.Run to run subtests (which is how Go does parameterized tests). t.Run takes in a name for the subtest, and fmt is used to format that name.

Reply:
Oh, I messed it with linter.